extension/src/experiments/index.ts

```diff
@@ -123,6 +130,14 @@ export class Experiments extends BaseRepository<TableData> {
       })
     )
 
+    this.dispose.track(
+      workspace.onDidChangeConfiguration((event: ConfigurationChangeEvent) => {
+        if (event.affectsConfiguration(ConfigKey.EXP_TABLE_HEAD_MAX_LAYERS)) {
+          this.cliData.update()
```

Comment:
Is there a better way to update the table when the configuration is updated 🤔

Reply:
No, I think this is the only way as we don't keep a copy of the exp show data.